recipes/centroid_rna_package/meta.yaml

```diff
@@ -0,0 +1,33 @@
+# building from source requiers also boost >= 1.4
+# this recipe uses working precompiled binaries
+package:
+  name: centroid_rna_package
+  version: 0.0.15
+
+build:
+  number: 0
+  skip: True # [osx]
+
+source:
+  sha1: e4318ffcd3bd4419a68c273fd924ec17a366c073
+  url: https://github.com/satoken/centroid-rna-package/releases/download/v0.0.15/centroid-rna-package-0.0.15-linux-x86_64.zip
+
+requirements:
+  run:
+    - viennarna >=1.8
```

Comment:
you need python as runtime requirement isn't it?

Reply:
No I don't need python at runtime.